src/helpers/Url/Url.js

```diff
@@ -159,7 +159,9 @@ export function flattenHTMLToAppURL(html) {
  * @returns {string} New URL with app
  */
 export function addAppURL(url) {
-  return `${settings.apiPath}${url}`;
+  return url.indexOf(settings.apiPath) === 0
```

Comment:
Is it relevant here to also use settings.internalApiPath? We use both of them, as frontend comunicates with the backend through settings.internalApiPath.